inputFiles/lagrangianContactMechanics/LagrangeContactBubbleStab_singleFracCompression_base.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" ?>
+
+<Problem>
+  <Solvers
+    gravityVector="{0.0, 0.0, 0.0}">
+
+    <SolidMechanicsLagrangeContactBubbleStab
+      name="AugmentedLagrangian"
```

Comment:
This is dumb, but should we rename this solver so if future people use it they dont get confused? Because SolidMechanicsLagrangeContactBubbleStab im assuming does not define an augmented lagrange solver

Reply:
yeah, this is = not augemented lagrange it's just the Lagrange multiplier plus bubble that solves directly the saddle point system.

Reply:
so I think in the input file we should just change the name field so that its not AugmentedLagrangian

Reply:
yeah, I ll change it.